lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1088,3 +1092,23 @@ def test_student_state(self, default_store):
         # Trigger the assertions embedded in the ViewCheckerBlocks
         response = views.index(request, unicode(course.id), chapter=chapter.url_name, section=section.url_name)
         self.assertEquals(response.content.count("ViewCheckerPassed"), 3)
+
+
+@patch.dict('django.conf.settings.FEATURES', {'ENABLE_MOBILE_XBLOCKS': True})
+class TestRenderXBlock(RenderXBlockTestMixin, ModuleStoreTestCase):
```

Comment:
There are no tests in here and i don't see any other test classes subclassing this?